model_interfaces/charlie_interface.py

```diff
@@ -135,3 +138,10 @@ def reset(self):
         )
 
         self.context = []
+
+
+    def load(self):
+        pass
+
+    def save(self):
```

Comment:
I believe this is the right implementation, because Charlie works through an online API, but it would be good to have that explained in a comment.